notebooks/concepts/Imviz_RadialPlot_Example.ipynb

```diff
@@ -0,0 +1,685 @@
+   "outputs": [],
+   "source": [
+    "# We can use the glue_app stuff here, but I just want to\n",
+    "# see if we get the same result in Matplotlib.\n",
+    "# Should look the same as \"Subset based selection\" below.\n",
+    "#\n",
+    "# Kyle, this is what will get sent to the frontend:\n",
+    "#      r.ravel(), img\n",
+    "#\n",
+    "# Frontend consideration: I don't think we can use\n",
+    "# glue_app.add_data() because we do not want to flood\n",
+    "# the app data collection with temporary buffer.\n",
+    "# Would be nice to have a plot-and-forget way to do this.\n",
+    "plt.plot(r.ravel(), img, 'k.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ec99610f",
+   "metadata": {},
+   "source": [
+    "# Subset based selection"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fcc6bfbc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "glue_app = imviz.app._application_handler\n",
+    "imdata = glue_app.data_collection[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "293fe111",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "imdata.subsets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8789a764",
+   "metadata": {},
+   "source": [
+    "Draw a rectangular subset around a star or whatever"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8c0ca41f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "subset0 = imdata.subsets[0]\n",
+    "subset0.components"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d3fe045b",
+   "metadata": {},
+   "source": [
+    "Assume the \"center\" is the middle pixel or between the two center-most pixels"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dd6febce",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x0 = np.mean(subset0['Pixel Axis 1 [x]'])\n",
+    "y0 = np.mean(subset0['Pixel Axis 0 [y]'])\n",
+    "x0, y0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f3471330",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dx = subset0['Pixel Axis 1 [x]'] - x0\n",
+    "dy = subset0['Pixel Axis 0 [y]'] - y0\n",
+    "r = np.hypot(dx, dy)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f44ba00e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# remove existing subset just so we can re-run this cell if desired\n",
+    "torem = [d for d in glue_app.data_collection if d.label =='plotdat_sub']\n",
+    "for d in torem:\n",
+    "    glue_app.data_collection.remove(d)\n",
+    "\n",
+    "glue_app.add_data(plotdat_sub=dict(rdat=r, dat=subset0['SCI,1']))\n",
+    "scat2d = glue_app.scatter2d(data='plotdat_sub', x='rdat', y='dat')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9c27908d",
+   "metadata": {},
+   "source": [
+    "# Live update from subset version "
```

Comment:
I don't know how I feel about this. Since we are attaching the radial plot to "simple aperture photometry" plugin, having live update would make aperture photometry calculations more complicated too. Maybe this is out of scope, @PatrickOgle, @orifox, or @camipacifici?